core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/eth2/Eth2SignForIdentifierHandler.java

```diff
@@ -188,29 +196,30 @@ private Bytes computeSigningRoot(final Eth2SigningRequestBody body) {
     switch (body.getType()) {
       case BLOCK:
         checkArgument(body.getBlock() != null, "block must be specified");
-        return SigningRootUtil.signingRootForSignBlock(
-            body.getBlock().asInternalBeaconBlock(), body.getForkInfo().asInternalForkInfo());
+        return signingRootUtil.signingRootForSignBlock(
+            body.getBlock().asInternalBeaconBlock(eth2Spec),
+            body.getForkInfo().asInternalForkInfo());
       case ATTESTATION:
         checkArgument(body.getAttestation() != null, "attestation must be specified");
-        return SigningRootUtil.signingRootForSignAttestationData(
+        return signingRootUtil.signingRootForSignAttestationData(
             body.getAttestation().asInternalAttestationData(),
             body.getForkInfo().asInternalForkInfo());
       case AGGREGATE_AND_PROOF:
         checkArgument(body.getAggregateAndProof() != null, "aggregateAndProof must be specified");
-        return SigningRootUtil.signingRootForSignAggregateAndProof(
+        return signingRootUtil.signingRootForSignAggregateAndProof(
             body.getAggregateAndProof().asInternalAggregateAndProof(),
             body.getForkInfo().asInternalForkInfo());
       case AGGREGATION_SLOT:
         checkArgument(body.getAggregationSlot() != null, "aggregationSlot must be specified");
-        return SigningRootUtil.signingRootForSignAggregationSlot(
+        return signingRootUtil.signingRootForSignAggregationSlot(
             body.getAggregationSlot().getSlot(), body.getForkInfo().asInternalForkInfo());
       case RANDAO_REVEAL:
         checkArgument(body.getRandaoReveal() != null, "randaoReveal must be specified");
-        return SigningRootUtil.signingRootForRandaoReveal(
+        return signingRootUtil.signingRootForRandaoReveal(
             body.getRandaoReveal().getEpoch(), body.getForkInfo().asInternalForkInfo());
       case VOLUNTARY_EXIT:
         checkArgument(body.getVoluntaryExit() != null, "voluntaryExit must be specified");
-        return SigningRootUtil.signingRootForSignVoluntaryExit(
+        return signingRootUtil.signingRootForSignVoluntaryExit(
             body.getVoluntaryExit().asInternalVoluntaryExit(),
             body.getForkInfo().asInternalForkInfo());
       case DEPOSIT:
```

Comment:
Is there an alternative to using the deprecated compute_signing_root method for computing the deposit signing root?